app/default/content_routes.py

```diff
@@ -27,15 +27,16 @@ def all_questions(fund_short_name, round_short_name):
         f"All questions page loaded for fund {fund_short_name} round"
         f" {round_short_name}."
     )
-    round = get_round_data_by_short_names(fund_short_name, round_short_name)
-    if not round:
-        return abort(404)
 
     if fund_short_name.lower() == "cof":
-        return render_template(
-            f"{fund_short_name.lower()}_{round_short_name.lower()[0:2]}_all_questions.html",
-            round_title=round.title,
-        )
+        if round := get_round(
+            fund_short_name=fund_short_name, round_short_name=round_short_name
```

Comment:
For consistency, these should probably be separated onto their own lines, as [seen below](https://github.com/communitiesuk/funding-service-design-frontend/pull/366/files#diff-25ffeea551d125947892e07d498380fe030e5f7c005ba15e165622de5d567f0eR242)